js/index.js

```diff
@@ -1,42 +1,153 @@
+
+// main content
+let topContentH4 = document.querySelectorAll('.top-content h4');
+topContentH4[0].innerText = siteContent['main-content']['features-h4'];
+topContentH4[1].innerText = siteContent['main-content']['about-h4'];
+
+let topContentP = document.querySelectorAll('.top-content p');
+topContentP[0].innerText = siteContent['main-content']['features-content'];
+topContentP[1].innerText = siteContent['main-content']['about-content'];
+
+let bottomContentH4 = document.querySelectorAll('.bottom-content h4');
+bottomContentH4[0].innerText = siteContent['main-content']['services-h4'];
+bottomContentH4[1].innerText = siteContent['main-content']['product-h4'];
+bottomContentH4[2].innerText = siteContent['main-content']['vision-h4'];
+
+let bottomContentP = document.querySelectorAll('.bottom-content p');
+bottomContentP[0].innerText = siteContent['main-content']['services-content'];
+bottomContentP[1].innerText = siteContent['main-content']['product-content'];
+bottomContentP[2].innerText = siteContent['main-content']['vision-content'];
+
+// contact
+let contactH4 = document.querySelector('.contact h4');
+contactH4.innerText = siteContent.contact['contact-h4'];
+
+let contactP = document.querySelectorAll('.contact p');
+contactP[0].innerText = siteContent.contact.address;
+contactP[1].innerText = siteContent.contact.phone;
+contactP[2].innerText = siteContent.contact.email;
+
+let footerP = document.querySelector('footer p');
+footerP.innerText = siteContent.footer.copyright;
+
+// Utilize .appendChild() and .prepend() to add two new items to the navigation system.
+let prependedItem = document.createElement('a');
+prependedItem.href = '#';
+prependedItem.innerText = 'Prepend';
+nav.prepend(prependedItem);
+
+let appendedItem = document.createElement('a');
+appendedItem.href = '#';
+appendedItem.innerText = 'Append';
+nav.appendChild(appendedItem);
+
+// Change the color of the navigation text to be green.
+navItems = document.querySelectorAll('nav a');
+navItems.forEach(item => (item.style.color = 'green'));
+
+let body = document.querySelector('body');
+body.style.transition = 'all 0.3s ease';
+
+// Dark / Light Mode Toggle
+ctaButton.addEventListener('click', function() {
+  body.classList.toggle('darkmode');
+  let paragraphs = document.querySelectorAll('p');
+  let h4s = document.querySelectorAll('h4');
+  let mainContent = document.querySelector('.main-content');
+
+  if (body.classList.contains('darkmode')) {
+    ctaButton.innerText = 'Light Mode';
+    body.style.background = '#313537';
+    h4s.forEach(h4 => (h4.style.color = 'white'));
+    paragraphs.forEach(p => (p.style.color = 'white'));
+    navItems.forEach(a => (a.style.color = 'white'));
+    ctaText.style.color = 'white';
+    mainContent.style.borderTop = '2px solid white';
+    mainContent.style.borderBottom = '2px solid white';
+    logo.style.filter = 'invert(100%)';
+    ctaImg.style.filter = 'invert(100%)';
+    middleImg.style.filter = 'invert(100%)';
```

Comment:
Really cool stuff here. In future projects where you don't have to do everything in JS you can make a class, style it in CSS and use `body.classList.toggle("darkmode")` or something similar